src/learn/README.md

```diff
@@ -2,29 +2,52 @@
+Designed as a scalable alternative to larger, more expensive chains the Cudos Network allows complex applications to run completely on-chain where that would've previously been prohibitively expensive or simply impossible due to gas limits. Cudos Network runs on a Delegated Proof of Stake (DPoS) consensus model, with Validators checking transactions and providing compute cycles for DApps on the network to consume.
+
+Developers, individuals and businesses will deploy next-generation smart contracts that leverage CosmWasm (and later EVM via Ethermint) with the security assurances of distributed ledger technology. This higher level of performance and security is possible thanks to the Cudos blockchain’s ability to leverage a strong community of Validators with a robust node infrastructure. This gives users of the Cudos Network a high level of confidence in the various enterprise and developer use cases that are subsequently deployed.
+
+What makes the Cudos Network stand out from other blockchains is the deep level of integration for high-performance compute capacity at the very core of the network. This allows Cudos to provide an extra computing layer to blockchains, unlocking an extensive array of additional use cases for which high-performance computing power is needed.
+
+### Ecosystem
+
+Cudos Network is built using Tendermint Core and inherits its Delegated Proof of Stake (DPoS) algorithm. In addition to leveraging Tendermint Core, the Cudos Network is supported by the Cosmos SDK to achieve high throughput, availability and security. This architecture allows easy cross-chain compatibility with many other blockchains, particularly those already built on the Cosmos SDK, by leveraging the nascent IBC protocol for moving assets between these systems securely.
 
-## What is CUDOS?
+The promise of computing, which is core to the Cudos Network’s mission, is also expanded beyond Cosmos-based blockchains through bespoke bridges and cryptographic approaches to cross-chain interoperability. Cudos Network launches with a bridge to Ethereum, and we plan to expand our compute offering to a wider array of networks in the future.
+
+### Design Principles
+
+1. Making efficient use of the world’s existing computing resources
+2. Multi-chain support
+3. Public, permission-less software > privately owned source code
+
+## CUDOS
 
 CUDOS is a multi-chain layer 2 solution, providing Turing-complete compute and external data to blockchains. Because blockchains are immutable, this means they are generally very limiting in what they can do. This very much restricts their use, and without another solution use cases like DeFi would not exist. CUDOS provides a bridge between the blockchain world and the outside world to enable blockchains to consume outside data, APIs and compute.
```

Comment:
I would actually remove everything on this page from this point on, as it either needs a rewrite because it's not accurate anymore, or repeats something said above in a different way.